tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -986,3 +1010,43 @@ func waitForFullSyncToFinish(client clientset.Interface, ctx context.Context,
 	})
 	return waitErr
 }
+
+// checkForEventWithMessage fetches events list of the given object name and checkes for
+// specified message and returns true if expected error message found
+func checkForEventWithMessage(client clientset.Interface, namespace string,
```

Comment:
`waitForEvent()` does same thing I think

Reply:
In waitForEvent(), it is waiting and polling for a particular event to occur, but in this case this is not required as I just need to check if this event exists or not.